static/app/utils/profiling/flamegraph/hooks/useFlamegraphColors.ts

```diff
@@ -1,21 +0,0 @@
-import {Flamegraph} from '../../flamegraph';
-import {FlamegraphFrame} from '../../flamegraphFrame';
-import {ColorChannels, FlamegraphTheme} from '../flamegraphTheme';
-
-export function useFlamegraphColors(
-  theme: FlamegraphTheme,
-  ...flamegraphs: Flamegraph[]
-): {colorBuffer: number[]; colorMap: Map<string | number, ColorChannels>} {
-  const frames = flamegraphs.reduce<FlamegraphFrame[]>((acc, flamegraph) => {
-    return acc.concat(flamegraph.frames);
-  }, []);
-
-  // Generate colors for the flamegraph
-  const {colorBuffer, colorMap} = theme.COLORS.STACK_TO_COLOR(
-    frames,
-    theme.COLORS.COLOR_MAP,
-    theme.COLORS.COLOR_BUCKET
-  );
-
-  return {colorBuffer, colorMap};
-}
```

Comment:
This was never used and this doesn't respect the color coding preferences. Just gonna delete it